SignalMessaging/environment/Release.m

```diff
@@ -14,11 +14,13 @@ @implementation Release
 
 + (Environment *)releaseEnvironment
 {
+    // Order matters here.
+    TSStorageManager *storageManager = [TSStorageManager sharedManager];
```

Comment:
Was something broken before? I can't think of any relevant side effects `[TSStorageManager sharedManager]` would have.

Reply:
There was a race wherein (if I recall correctly) `OWSContactsManager`'s initializer kicks of `loadCachedDisplayNames` which async uses `TSStorageManager`, which expects to be initialized on the main thread.  Something like that.